packages/models/src/item.ts

```diff
@@ -1,8 +1,10 @@
+import { Descriptor } from "./descriptor";
+
 export interface Item {
   name: string;
   url: string;
   descriptorUrl: string;
-  desriptorDom: Object;
+  descriptorDom: Descriptor | null;
```

Comment:
Shouldn't be necessary to explicitly add `null` as a type. Did `descriptorDom: Descriptor` give any trouble?